pipeline-runner/R/qc-7-embed_and_cluster.R

```diff
@@ -3,63 +3,90 @@
 # Compute embedding step where we run dimensional reduction technniques such as t-SNE and  UMAP. Moreover, the cluster analysis is
 # done also in this step.
 
-embed_and_cluster <- function(scdata, config, sample_id, cells_id, task_name = "configureEmbedding") {
-  message("starting clusters")
-  clustering_method <- config$clusteringSettings$method
-  methodSettings <- config$clusteringSettings$methodSettings[[clustering_method]]
-  message("Running clustering")
-  cellSets <- runClusters(clustering_method, methodSettings$resolution, scdata)
-  message("formatting cellsets")
-  formated_cell_sets <- format_cell_sets_object(cellSets, clustering_method, scdata@misc$color_pool)
-  message("udating through api")
+embed_and_cluster <-
```

Comment:
it looks like nothing got changed in this file apart from formatting -- is that right?

Reply:
that is correct